sdk/typescript/src/signers/txn-data-serializers/local-txn-data-serializer.ts

```diff
@@ -150,10 +150,9 @@ export class LocalTxnDataSerializer implements TxnDataSerializer {
         break;
       case 'moveCall':
         const moveCall = unserializedTxn.data as MoveCallTransaction;
-        const pkg = await this.provider.getObjectRef(moveCall.packageObjectId);
         tx = {
           Call: {
-            package: pkg!,
+            package: moveCall.packageObjectId,
```

Comment:
Unfortunately, we have to make this change backward compatible with older version of the network(e.g.., the upcoming TestNet 0.22.x). We would need to fetch the RPC version again and use ID or reference based on the version